api/src/main/resources/workbench.yaml

```diff
@@ -1979,6 +2222,130 @@ definitions:
         format: int64
         description: the count for this criteria type in the cohort
 
+  CreateCohortAnnotationDefinitionRequest:
+    type: object
+    required:
+      - name
+      - annotationType
+    properties:
+      name:
+        type: string
+        description: the name of this annotation.
+      annotationType:
+        $ref: "#/definitions/AnnotationType"
+
+  AnnotationType:
+    type: string
+    description: possible values representing the type of annotation.
+    enum: [STRING, BOOLEAN, INTEGER, DATE, TIMESTAMP]
```

Comment:
Shouldn't enum be an option here? Are you doing that later?

Reply:
It could be but Jon Scherdin doesn't have a use case for enum in the current app. So I'm excluding it for now, but should be easy enough to add if we feel it is necessary.